snapshot-cross-account-replicator/functions/create_snapshots.py

```diff
@@ -0,0 +1,31 @@
+import boto3
+import botocore
+import datetime
+import os
+
+instances = os.environ['RDS_INSTANCE_IDS']
+setup_name = os.environ['SETUP_NAME']
+
+
+def lambda_handler(event, context):
+    """Lambda entry point"""
+
+    print('Lambda function start: going to create snapshots for the RDS instances ' + instances)
+
+    source = boto3.client('rds')
+
+    for instance in instances.split(','):
+        now = datetime.datetime.now()
+        db_snapshot_name = instance + '_' + now.strftime('%Y-%m-%d-%H-%M')
+        try:
+            source.create_db_snapshot(
```

Comment:
why create new snapshots instead of re-using the automated snapshots?

Reply:
See the discussion in the PC issue https://github.com/skyscrapers/engineering/issues/531#issuecomment-929218057